CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# v3.10.0
+
+Reaction v3.10.0 adds minor features and performance enhancements, and contains no breaking changes since v3.9.0
```

Comment:
I think either in this heading or under notable changes we should mention that node's ES Modules support is stable but still experimental and while we are not aware of any specific breaking changes or issues, some CommonJS/ES Modules refactoring may be necessary in some installations.

Reply:
What would need to be changed, if we supported it in its experimental form before, do any changes here make it so you can or can't use it any differently than 3.9?

I'm fine calling it out, but just wondering if nothing changes officially, do we need to call it out?

Reply:
I think I'm just nervous that this may be one of those "supposed to be compatible" upgrades but in the real world does cause breakages. This is mostly anxiety on my part. Going to mark this resolved so things can progress.

Reply:
That's fair. It doesn't hurt to add extra info, was just wondering. I'll add it.